### ISSUE TITLE
"vm.install.path" Server Parameter doesn't work - WIN10

### ISSUE THREAD

**Issue by davigre** [redhat-developer/vscode-server-connector#457]
I use Wildfly-10.0.0.Final with JDK_8_251.

All was working ok, but now [redhat-developer/vscode-java](https://github.com/redhat-developer/vscode-java) needs JDK 11+ so I change java.home to `C:\\Program Files\\AdoptOpenJDK\\jdk-11.0.8.10-hotspot`.

The problem is that 'Server Connector' normally tries to run Wildfly with 'java.home' JDK. 

Reading the [manual](https://github.com/redhat-developer/vscode-rsp-ui#global-server-parameters) I add `"vm.install.path": "C:\\Program Files\\Java\\jdk1.8.0_251"` to run Wildfly with JDK8, but the following exception is output as soon as I save the configuration:

```
The server management server is running on port 8500

21:58:47.120 [onPool-worker-5] ERROR o.j.t.r.s.w.s.AbstractJBossServerDelegate:397 - 
org.jboss.tools.rsp.eclipse.core.runtime.CoreException: 
	at org.jboss.tools.rsp.server.spi.launchers.AbstractJavaLauncher.getLaunchCommand(AbstractJavaLauncher.java:72)
	at org.jboss.tools.rsp.server.wildfly.servertype.AbstractJBossServerDelegate.updateDependentAttributes(AbstractJBossServerDelegate.java:395)
	at org.jboss.tools.rsp.server.wildfly.servertype.AbstractJBossServerDelegate.updateServer(AbstractJBossServerDelegate.java:384)
	at org.jboss.tools.rsp.server.wildfly.servertype.impl.WildFlyServerDelegate.updateServer(WildFlyServerDelegate.java:69)
	at org.jboss.tools.rsp.server.model.ServerModel.updateServer(ServerModel.java:780)
	at org.jboss.tools.rsp.server.ServerManagementServerImpl.updateServerSync(ServerManagementServerImpl.java:391)
	at org.jboss.tools.rsp.server.ServerManagementServerImpl.lambda$13(ServerManagementServerImpl.java:387)
	at org.jboss.tools.rsp.server.ServerManagementServerImpl.lambda$36(ServerManagementServerImpl.java:906)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

and when I try to 'start the server':

`Unable to start the server:  Extension backend error - server can not be started: server gtllocal can not find a valid virtual machine to use.`

And when I reopen vscode in output 'Red HatServer Connector (stdout)':

```
08:36:40.350 [unch RSP Server] ERROR o.j.t.r.s.w.s.AbstractJBossServerDelegate:397 - 
org.jboss.tools.rsp.eclipse.core.runtime.CoreException: 
	at org.jboss.tools.rsp.server.spi.launchers.AbstractJavaLauncher.getLaunchCommand(AbstractJavaLauncher.java:72)
	at org.jboss.tools.rsp.server.wildfly.servertype.AbstractJBossServerDelegate.updateDependentAttributes(AbstractJBossServerDelegate.java:395)
	at org.jboss.tools.rsp.server.wildfly.impl.ExtensionHandler$PostWildFlyCreationListener.serverAdded(ExtensionHandler.java:104)
	at org.jboss.tools.rsp.server.model.ServerModel.fireServerAdded(ServerModel.java:419)
	at org.jboss.tools.rsp.server.model.ServerModel.addServer(ServerModel.java:414)
	at org.jboss.tools.rsp.server.model.ServerModel.loadServers(ServerModel.java:196)
	at org.jboss.tools.rsp.server.model.ServerModel.loadServers(ServerModel.java:173)
	at org.jboss.tools.rsp.server.model.ServerPersistenceManager.loadState(ServerPersistenceManager.java:91)
	at org.jboss.tools.rsp.server.ServerManagementServerLauncher.launch(ServerManagementServerLauncher.java:93)
	at org.jboss.tools.rsp.server.ServerCoreActivator.lambda$1(ServerCoreActivator.java:58)
	at java.base/java.lang.Thread.run(Thread.java:834)

The server management server is running on port 8500
```

and in 'Red Hat Server Connector (stderr)':

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.felix.framework.ext.ClassPathExtenderFactory$DefaultClassLoaderExtender (file:/C:/Users/david/.vscode/extensions/redhat.vscode-server-connector-0.23.2/server/bin/felix.jar) to method java.net.URLClassLoader.addURL(java.net.URL)
WARNING: Please consider reporting this to the maintainers of org.apache.felix.framework.ext.ClassPathExtenderFactory$DefaultClassLoaderExtender
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by lstocchi** [comment]
Thanks for opening this issue @davigre. I was able to replicate this. I'll give it a look asap.

**Comment by lstocchi** [comment]
@davigre we just released the new server connector extension (0.23.3) which fixes the issue with the `vm.install.path` property. Now you should be able to set it up and use rsp-server with the latest vscode java.
Let us know if you face any other issue. Thank you and sorry for taking so long.

**Comment by davigre** [comment]
@lstocchi ,

Thanks a lot. I already tested.
